src/ui/public/template_vis_type/template_vis_type.js

```diff
@@ -1,23 +1,23 @@
 import _ from 'lodash';
-import VisVisTypeProvider from 'ui/vis/vis_type';
-import TemplateVisTypeTemplateRenderbotProvider from 'ui/template_vis_type/template_renderbot';
-export default function TemplateVisTypeFactory(Private) {
-  var VisType = Private(VisVisTypeProvider);
-  var TemplateRenderbot = Private(TemplateVisTypeTemplateRenderbotProvider);
+define(function (require) {
```

Comment:
Like the other file, this file should continue to use es6 modules rather than AMD.